### DISCUSSION TITLE
cockpit  not showing adminstrative access

### DISCUSSION THREAD

**Discussion by Kumar6295** [cockpit-project/cockpit#17746] [Q&A] [answered]
### Explain what happens

Hi,
We are using cockpit below version.
Version 215.
Licensed under: [GNU LGPL version 2.1 1]

Please find the Attachement instead of administrative access , it is showing privileged , we are unable to click it.
![cockpit](https://user-images.githubusercontent.com/72788032/191437048-1d03498f-5e2d-424f-98eb-6451b60a2653.png)

we try to install cockpit in ubuntu and centos both the servers same issue. it is not showing adminstrative access.

thanks,
kumar

### Version of Cockpit

 Version 215. Licensed under: [GNU LGPL version 2.1 1]

### Server operating system

Ubuntu

### Server operating system version

20.04

### What browsers are you using?

Chrome

**Comment by marusak** [comment, marked as answer]
> instead of administrative access , it is showing privileged

You are using version 215. Since then we have done significant changes how this operation works and looks. This is expected behavior on this version.

> we are unable to click it

Correct. I admittedly don't remember anymore how this used to work, but I think you would need to log out and use checkbox to get into "privileged" mode. Also you are logged in as root so this does not change much anyway. Out of interest, why would you want to click it? It is just static information. New version has this as button and you could drop privileges, but does that make sense with root user?

**Reply by Kumar6295** [reply]
ok,
Actually we need to disable/hide the termianl/account/networking tabs to particular user(not root user) in cockpit UI.
How we can achieve it.

Thanks,
Kumar

**Reply by marusak** [reply]
See here https://github.com/cockpit-project/cockpit/issues/9299

To sum up, you can do it, but it does not prevent these users from executing any arbitrary commands.
